README.md

```diff
@@ -89,15 +87,14 @@ test('the title is welcoming', function(assert) {
 A basic codemod to automatically convert your assertions is available at
 [https://github.com/simplabs/qunit-dom-codemod](https://github.com/simplabs/qunit-dom-codemod).
 
-
 ### TypeScript
 
 `qunit-dom` includes type definition files, but the way it extends QUnit means
 that you need import it somewhere so that TS and your editor can pick up the
 types. It is recommended to add the following line to your
 `tests/test-helper.ts` file:
 
-```
+```bash
```

Comment:
these changes appear to be unrelated to the proposed feature, no?

Reply:
+1, I'd remove these.

Reply:
Removed